src/server/page_server.hpp

```diff
@@ -198,8 +198,12 @@ class page_server
 	follower_connection_handler &operator= (const connection_handler &) = delete;
 	follower_connection_handler &operator= (connection_handler &&) = delete;
 
+	~follower_connection_handler ();
```

Comment:
Nit, I find these names quite confusing:
- `follower_connection_handler` -> `from_follower_connection_handler` (ie: connection from the followerr) or `as_followed_connection_handler` (ie: "acting as followed")
- `followee_connection_handler` -> `to_followed_connection_handler` (ie: connection to the followed) or 'as_follower_connection_handler` (ie: "acting as follower")

Whichever your prefer, or even some other naming scheme which is more explanatory than the current.

Reply:
I agree the naming is somewhat confusing and actually struggled to come up with a proper one. It seems that your suggestion is better. I'll create an issue to modify the name. In the issue, I'll modify the names to yours or something more explanatory.